tcf_website/static/course/javascript/instr_data.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Dropdown data population for the course-instructor page
+ * Author: j-alicia-long, 12/13/2020
+*/
+
+// Executed when DOM is ready
+jQuery(function($) {
+
+    /* Fetch instructor data for given course */
+    var pageSize = "1000";
+    var instrEndpoint = `/api/instructors/?course=${courseID}` +
+        `&page_size=${pageSize}`;
+    // var courseID is from global var in template
+    $.getJSON(instrEndpoint, function(data) {
+        console.log(data);
+        // Generate dropdown links
+        $.each(data.results, function(i, instr) {
+            $("<a />", {
+                id: `instr-${instr.id}`,
+                class: "dropdown-item",
+                href: `/course/${courseID}/${instr.id}`,
+                text: instr.first_name + " " + instr.last_name
+            }).appendTo("#instructor");
+        });
+        return this;
+    })
+    .done(function() {
+      // Enable instructor selector
+      $("#instructor").prop("disabled", false);
+    });
+});
```

Comment:
Would it be a lot of extra work (e.g. changes to the API) to add the `All` aggregation of instructors?

Reply:
Edit: realized there would have to be an "all" page to implement this

Reply:
yeahh we'd have to add tweaks to the grade data & display for that page so might be out of scope